### ISSUE TITLE
Adding more support for swift

### ISSUE THREAD

**Issue by AncAinu** [Tricertops/KeepLayout#76]
Hello Martin,

As you know I like very much KeepLayout, and since swift 2.0, it's not so nice to have to use KeepRequired etc.
So I decided to throw myself into it and I did a little support for KeepLayout specially for Swift:
https://github.com/AncAinu/SwiftKeepLayout

If you think it's a nice idea, you could include a little link into your readme, it's a full pod easy to install.
If you don't, well, I don't mind ^^

Hope you find it nice :)

(I will probably enhance it over time by using it and see if I can make it even better, maybe even fully translate KeepLayout to swift one day if you don't :stuck_out_tongue: )

**Comment by Tricertops** [comment]
Hi, nice job! I look forward to your improvements. I don’t use Swift as main tool yet, so I didn’t spend much time with optimizing Swift experience. I’m afraid that any really good Swift improvement would break Objective-C compatibility, and that’s my priority. I can imagine the Swift branch to evolve in parallel to the Obejctive-C branch in future.

**Comment by Tricertops** [comment]
I just pushed a Swift compatibility API and merged `swift-legacy` branch into `master`. It is now possible to use KeepLayout framework from mixed projects:

```objc
view.keepWidth.equal = 100;
```
```swift
view.keepWidth.equal = 100
```
